backend/substrapp/kubernetes_utils.py

```diff
@@ -47,6 +47,14 @@ def get_security_context(root: bool = False, capabilities: list[str] = None) ->
     return security_context
 
 
+def get_resources_requirements(
+    *, cpu_request: str = "1000m", memory_request: str = "100M", memory_limit: str = "1G"
```

Comment:
And no CPU limit?

Reply:
We don't want to set any CPU limits on CPU-heavy tasks, to avoid CPU throttling
In addition, GPU can't be shared across tasks, so compute functions will be scheduled on separate VM, and it makes no sense to limit CPU in that setting.

Reply:
I meant, we are setting pretty low defaults in comparison to args passed where that function are called, so I would assume we would provide a default for CPU limit to prevent any misuse of the function (that can be eventually cancelled for CPU-heavy tasks). But as this scenario is a bit unlikely to happen, no problem to keep it as is 👍